.travis.yml

```diff
@@ -1,82 +1,50 @@
 sudo: true
-language: generic
+language: haskell
```

Comment:
Let's also add one line here with the following content:

```yaml
dist: trusty
```

I guess this allows to fetch GHC versions from hvr PPA